internal/testutil/acc/cluster.go

```diff
@@ -18,6 +18,7 @@ type ClusterRequest struct {
 
 type ClusterInfo struct {
 	ProjectIDStr        string
+	ProjectID           string
```

Comment:
not sure what this change is for?  It looks like we pass ProjectID as a param to GetClusterInfo and returns it without doing any else